diff --git a/test/spec_tests.py b/test/spec_tests.py
index 194c510..8931ef7 100755
--- a/test/spec_tests.py
+++ b/test/spec_tests.py
@@ -104,15 +104,14 @@ def get_tests(specfile):
                 state = 0
                 example_number = example_number + 1
                 end_line = line_number
-                if 'disabled' not in extensions:
-                    tests.append({
-                        "markdown":''.join(markdown_lines).replace('→',"\t"),
-                        "html":''.join(html_lines).replace('→',"\t"),
-                        "example": example_number,
-                        "start_line": start_line,
-                        "end_line": end_line,
-                        "section": headertext,
-                        "extensions": extensions})
+                tests.append({
+                    "markdown":''.join(markdown_lines).replace('→',"\t"),
+                    "html":''.join(html_lines).replace('→',"\t"),
+                    "example": example_number,
+                    "start_line": start_line,
+                    "end_line": end_line,
+                    "section": headertext,
+                    "extensions": extensions})
                 start_line = 0
                 markdown_lines = []
                 html_lines = []
